internal/core/command/device.go

```diff
@@ -164,9 +165,32 @@ func getCommands(ctx context.Context) (int, []models.CommandResponse, error) {
 			return http.StatusInternalServerError, nil, err
 		}
 	}
-	var cr []models.CommandResponse
+
+	if len(devices) == 0 {
+		return http.StatusOK, cr, err
+	}
+
+	dps, err := mdpc.DeviceProfiles(ctx)
```

Comment:
Why do we need to load device profiles here when the command service itself knows what commands are available for a device?

`http://localhost:48082/api/v1/device/name/Random-Integer-Generator01`
`http://localhost:48082/api/v1/device/{UUID}`